e2e/dashboard/test_dashboard_navigation.py

```
@@ -0,0 +1,190 @@
+            print("  - dashboard_09_final.png")
+
+            return True
+
+        except AssertionError as e:
+            print(f"\n[ASSERTION ERROR] {e}")
+            page.screenshot(path="/tmp/dashboard_error_assertion.png")
+            import traceback
+
+            traceback.print_exc()
+            return False
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)